dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -72,6 +72,8 @@ def send_email_to_brief_user_via_notify(notify_client, notify_template_id, user,
                 notify_client.send_email(
                     user['emailAddress'], notify_template_id, email_context_data, allow_resend=False
                 )
+        except EmailTemplateError:
+            raise  # do not try to continue
```

Comment:
Why do we need to explicitly re-raise this?

Reply:
Because it's a subclass of EmailError, if we don't catch and re-raise it will be handled by the EmailError handler.